stdlib/std/cells.tact

```diff
@@ -191,3 +201,16 @@ inline fun emptyCell(): Cell {
 inline fun emptySlice(): Slice {
     return emptyCell().asSlice();
 }
+
+struct DataSize {
+    cells: Int;
+    bits: Int;
+    refs: Int;
+}
+
+asm fun computeDataSize(cell: Cell, maxCells: Int): DataSize { CDATASIZE TRIPLE }
```

Comment:
Would `TRIPLE` work here? Structures are returned as tensors, right?

Reply:
`TRIPLE` makes three elements into a tuple of length 3. I think it should work

Reply:
yes, but our structs are tensors, not tuples, right?

Reply:
ah, I must have mistaken it with the unbundled approach in the storage